recipes/porechop/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: porechop
+  version: '0.2.3_seqan2.1.1'
+
+source:
+  fn:  Porechop-0.2.3_seqan2.1.1.tar.gz
+  url: https://github.com/jvolkening/Porechop/archive/v0.2.3_seqan2.1.1.tar.gz
+  md5: 648fbf44b8fe70aa771786b5180b4e22
+
+build:
+  number: 0
+  skip: True # [py27]
+
+requirements:
+  build:
+    - python >=3.4
+    - gcc  # [not osx]
```

Comment:
I'm wondering why seqan is not a dependency, and if its only shipping header, if you really need to encode it in the version string?

Reply:
The underlying problem here is the general one of conda not having a recent GCC supporting `c++14`. To compile Porechop for bioconda, it is therefore necessary to downgrade SeqAn to the last release buildable with GCC 4.8, which is SeqAn 2.1.1.

The only difference between my forked branch and mainstream Porechop is the downgraded SeqAn. That is why I encode this information in the version string -- it doesn't seem quite right to just use the version from upstream. If you have another suggestion I'm happy to listen.

As for making SeqAn a dependency, that's not how it's done in upstream Porechop so I didn't see a reason to change it. The alternative would be to modify upstream to make SeqAn a dependency. Either way it requires a modified version.

Feel free to push whatever modifications you wish to this. I'm just trying to find a way to make it work. Hopefully a recent GCC will make it into conda and then all of this can go away.

Reply:
Ok, got it. Can you please write this into the recipe as a comment. Also that the URL needs to be changed as soon as we get a new compiler?